requirements.txt

```diff
@@ -5,9 +5,10 @@ canonicalwebteam.discourse==5.4.2
 canonicalwebteam.image-template==1.3.1
 canonicalwebteam.store-api==4.4.0
 canonicalwebteam.docstring-extractor==1.2.0
-canonicalwebteam.store-base==0.0.7
+canonicalwebteam.store-base==0.0.8
+
 Flask-WTF==1.0.1
-humanize==4.4.0
```

Comment:
Yes, it's a conflict. I tried upgrading in storebase but it conflicted with another dependency.

Reply:
It would be good to investigate if there is a way to upgrade store-base and it's conflicts. I don't think we should downgrade dependencies in live sites unless we're 100% sure we have to?